README.md

```diff
@@ -527,6 +530,14 @@ __AWESOME__ apps counter: __130__ 🎉
 - [x] [GitHub](https://github.com/syncthing/syncthing-android)
 - [x] [Official page](https://syncthing.net/)
 
+### Syncthing-Fork
```

Comment:
Just more features, that's all?

Reply:
much more convinience 
it is worth using this app instead of official one as it suck when a transer of 10GB stops in betweeen and you not near youe mobile